inky/opts.go

```diff
@@ -12,27 +12,27 @@ import (
 )
 
 var (
-	displayVariantMap = map[int]string{
-		0:  "",
-		1:  "Red pHAT (High-Temp)",
-		2:  "Yellow wHAT",
-		3:  "Black wHAT",
-		4:  "Black pHAT",
-		5:  "Yellow pHAT",
-		6:  "Red wHAT",
-		7:  "Red wHAT (High-Temp)",
-		8:  "Red wHAT",
-		9:  "",
-		10: "Black pHAT (SSD1608)",
-		11: "Red pHAT (SSD1608)",
-		12: "Yellow pHAT (SSD1608)",
-		13: "",
-		14: "7-Colour (UC8159)",
-		15: "7-Colour 640x400 (UC8159)",
-		16: "7-Colour 640x400 (UC8159)",
-		17: "Black wHAT (SSD1683)",
-		18: "Red wHAT (SSD1683)",
-		19: "Yellow wHAT (SSD1683)",
+	displayVariantMap = []string{
```

Comment:
Fun fact: you can make the slice an array with using `[...]` instead of `[]`.

I don't think the () are useful but it's not a big deal.

Reply:
Changed it to an array. 

The () might be handy to distinguish boards with different IC drivers tbh. I haven't checked but there might be even material differences between them and the existing code may not work for new ones (they are 2 different implementations at https://github.com/pimoroni/inky ATM)